src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,120 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataInsertException;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+
+    @Override
+    public Driver create(Driver driver) {
+        String insertDriverRequest = "INSERT INTO drivers (name,licenseNumber) values(?,?);";
```

Comment:
add space between commas

Reply:
rename to insertRequest we don't need driver here as it is known that we are working with drivers and our name will be shorter

Reply:
done